docs/application/native/guides/internationalization/i18n.md

```diff
@@ -1984,6 +1984,10 @@ To generate and manage an alphabetic index:
    ```
    i18n_alpha_idx_create("en", "US", &alpha_idx);
    ```
+   > **Note**  
+   > From Tizen 5.0 this function is deprecated. Instead use `i18n_alpha_idx_create_from_locale_id()`
```

Comment:
1) Why are we mentioning "Previously", is it to say if you the developer had previously set the country as US and language as en, then use this new locale ID?

2) Add a empty > tag after **Note**

Suggestions to change: (hope selected/set is correct in this context?)

This function is deprecated from Tizen 5.0. Instead, you can use `i18n_alpha_idx_create_from_locale_id()`
, which takes the locale ID parameter. Previously, if you have selected the language as `en` and country as `US`, then use the locale ID will be `en_US`.

OR 

This function is deprecated from Tizen 5.0. Instead. you can use `i18n_alpha_idx_create_from_locale_id()`
, which takes the locale ID parameter.  To set the language as `en` and country as `US`, use the locale ID `en_US`.

Reply:
I used second type of message and changed messages which do not fit to function role. To keep consistency in whole file, I leave notes without additional empty > tag, because at the end of each > **Note** are 2 additional spaces.